### ISSUE TITLE
Minimum Microsoft 365 Apps version for autosave and co-authoring?

### ISSUE THREAD

**Issue by TerryJ123** [MicrosoftDocs/microsoft-365-docs#8239]
The MIP documentation indicates that co-authoring should be available for Microsoft 365 Apps version 2107 and above, however, the Office release notes don't show this feature until 2202. Which is correct? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bed8ccbd-b955-4a11-f4dd-3a70b549a5f7
* Version Independent ID: 257024d1-afb3-954f-c848-21549cb64fac
* Content: [Enable co-authoring for documents encrypted by sensitivity labels in Microsoft 365 - Microsoft 365 Compliance](https://docs.microsoft.com/en-us/microsoft-365/compliance/sensitivity-labels-coauthoring?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/sensitivity-labels-coauthoring.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/sensitivity-labels-coauthoring.md)
* Service: **o365-seccomp**
* GitHub Login: @cabailey
* Microsoft Alias: **cabailey**

**Comment by cabailey** [comment]
@TerryJ123  Please provide a link to the release notes you're referencing & I'll investigate. Thanks.

**Comment by TerryJ123** [comment]
Thanks Carol

We had a client that was unable to get co-authoring and autosave working with 2107, however after the client updated Office to Version 2202 (Build 14931.20132) the autosave and co-authoring is now enabled. This is indicated in the release notes for Microsoft 365 apps - https://docs.microsoft.com/en-us/officeupdates/semi-annual-enterprise-channel-preview#version-2202-march-08

***@***.***

This does not match the MIP documentation - https://aka.ms/mipcoauth - which indicates it should work in 2107
***@***.***


[A picture containing logo  Description automatically generated]
Terry Johnson | Synergy Technical
804-469-0372| ***@***.***
www.synergy-technical.com<http://www.synergy-technical.com/>

**Comment by cabailey** [comment]
Mystery solved - this page actually says "Minimum version 2107 from Current Channel or Monthly Enterprise Channel" and if you check the supported versions table (https://docs.microsoft.com/en-us/microsoft-365/compliance/sensitivity-labels-office-apps?view=o365-worldwide#sensitivity-label-capabilities-in-word-excel-and-powerpoint) it lists 2107 for Current Channel & Monthly Enterprise Channel, and 2202 for Semi-Annual Enterprise Channel.

In a nutshell, the different channels have different minimum supported versions. I wasn't aware that 2202 was now released for Semi-Annual Enterprise Channel - I'll update the page accordingly.  Thanks for flagging!

**Comment by TerryJ123** [comment]
Thank you.
I know see that there is a note in the documentation that indicates it wasn't available for the Semi-annual channel. We missed that!

***@***.***
I guess that can be removed now.


[A picture containing logo  Description automatically generated]
Terry Johnson | Synergy Technical
804-469-0372| ***@***.***
www.synergy-technical.com<http://www.synergy-technical.com/>

**Comment by cabailey** [comment]
Yes, all fixed now!  Closing.

**Comment by TerryJ123** [comment]
Thank you Carol!

[A picture containing logo  Description automatically generated]
Terry Johnson | Synergy Technical
804-469-0372| ***@***.***
www.synergy-technical.com<http://www.synergy-technical.com/>
